### ISSUE TITLE
trac 1.4 support

### ISSUE THREAD

**Issue by ch-f** [trac-hacks/trac-wikinotification#16]
To support Trac 1.4 Trac Notification API needs to be updated. See https://trac.edgewall.org/wiki/TracDev/Proposals/AdvancedNotification/NotificationApi

For upcoming Trac 1.6, ITemplateStreamFilter needs to be ported to IRequestFilter.post_process_request, see https://trac.edgewall.org/wiki/TracDev/PortingFromGenshiToJinja#ReplacingITemplateStreamFilter

**Comment by rjollos** [comment]
> For upcoming Trac 1.6, ITemplateStreamFilter needs to be ported to IRequestFilter.post_process_request

Yeah, you can probably add the contextual navigation item through `post_process_request`.
